contributed_definitions/nyaml/NXxps.yaml

```diff
@@ -0,0 +1,474 @@
+        depends_on:
+          exists: recommended
+          doc: |
+            Reference to the transformation describing the orientation of the analyzer
+            relative to a defined coordinate system.
+        transformations(NXtransformations):
+          exists: recommended
+          analyser_take_off_polar_angle(NX_NUMBER):
+            unit: NX_ANGLE
+            doc: |
+              Polar tilt of the analyser with respect to the upward z-direction, defined by
+              the sample stage.
+            \@transformation_type:
+              enumeration: [rotation]
+            \@vector:
+              enumeration: [[1, 0, 0]]
+            \@depends_on:
+              enumeration: [analyser_take_off_azimuth_angle]
+          analyser_take_off_azimuth_angle(NX_NUMBER):
+            unit: NX_ANGLE
+            doc: |
+              Azimuthal tilt of the analyser from the y-direction towards the operator,
+              defined by the sample stage.
+            \@transformation_type:
+              enumeration: [rotation]
+            \@vector:
+              enumeration: [[0, 0, -1]]
+            \@depends_on:
+              enumeration: [/entry/sample_stage_cs_transformations/vamas_sample_stage_coordinate_system]
+    (NXprocess_mpes):
+      energy_referencing(NXcalibration):
+        exists: recommended
+      transmission_correction(NXcalibration):
+        exists: recommended
+    (NXsample):
+      depends_on:
+        exists: recommended
+        doc: |
+          Reference to the transformation describing the orientation of the sample
+          relative to a defined coordinate system.
+      transformations(NXtransformations):
+        exists: recommended
+        sample_rotation_angle(NX_NUMBER):
+          unit: NX_ANGLE
+          doc: |
+            Clockwise rotation about the sample normal.
+          \@transformation_type:
+            enumeration: [rotation]
+          \@vector:
+            enumeration: [[0, 0, -1]]
+          \@depends_on:
+            enumeration: [sample_normal_polar_angle]
+        sample_normal_polar_angle_of_tilt(NX_NUMBER):
+          unit: NX_ANGLE
+          doc: |
+            Polar tilt of the sample with respect to the upward z-direction, defined by the
+            sample stage.
+          \@transformation_type:
+            enumeration: [rotation]
+          \@vector:
+            enumeration: [[1, 0, 0]]
+          \@depends_on:
+            enumeration: [sample_normal_tilt_azimuth_angle]
+        sample_normal_tilt_azimuth_angle(NX_NUMBER):
+          unit: NX_ANGLE
+          doc: |
+            Azimuthal tilt of the sample from the y-direction towards the operator, defined
+            by the sample stage.
+          \@transformation_type:
+            enumeration: [rotation]
+          \@vector:
+            enumeration: [[0, 0, -1]]
+          \@depends_on:
+            enumeration: [/entry/sample_stage_cs_transformations/vamas_sample_stage_coordinate_system]
```

Comment:
Again strange naming and strange order, but maybe that is me coming from the ARPES world...

Reply:
see above